src/main/java/com/minecrafttas/tasmod/TASmod.java

```diff
@@ -81,18 +77,6 @@ public void preInit(FMLPreInitializationEvent ev) throws Exception {
 		} else if (Loader.isModLoaded("lotas")) {
 			throw new ModIncompatibleException("\n========================================================================\n" + "\n" + "Detected LoTAS to be loaded. TASMod and LoTAS are incompatible!\n" + "\n" + "========================================================================");
 		}
-		for (ModContainer mod : Loader.instance().getModList())
-        {
-            if(mod.getModId().equals("forge")) {
-            	String [] versionsplit=mod.getVersion().split("\\.");
-            	int forgeversion=Integer.parseInt(versionsplit[3]);
-            	if(MCVERSION.equals("1.12.2")&&forgeversion<2857) {
-            		throw new SecurityException("\n========================================================================\n" + "\n" + "The forge version you are using contains a critical Log4J exploit: 14.23.5." +forgeversion+ "\n"
-            				+ "Please update the forge version to something above 14.23.5.2857.\n"
-            				+ "You can find new versions under https://files.minecraftforge.net/net/minecraftforge/forge/index_1.12.2.html\n\n ========================================================================");
-            	}
-            }
```

Comment:
I see you are stealthily trying to remove my forge warning... Well we are switching to fabric anyway...

Reply:
nop, minecraft's 1.12.2.json was changed to use a new log4j version, therefore this warning is not required and just annoying.